packages/manager/core/app/src/pages/Details.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Tabs, TabList, TabPanels, Tab } from '@chakra-ui/react';
+import { useTranslation } from 'react-i18next';
+import { Outlet, NavLink } from 'react-router-dom';
+
+export default function DetailsPage(): JSX.Element {
+  const { t } = useTranslation('details');
+
+  const tabs = [
+    {
+      name: 'general',
+      title: t('tab_general'),
+      to: '',
+    },
+    {
+      name: 'nodes',
+      title: t('tab_nodes'),
+      to: 'nodes',
+    },
+  ];
+
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  return (
+    <Tabs
```

Comment:
you want to use the isLazy option to avoid making network calls for tabs that are not displayed, see : https://chakra-ui.com/docs/components/tabs#lazily-mounting-tab-panels